.github/workflows/release.yml

```diff
@@ -0,0 +1,59 @@
+name: Release
+
+on:
+  push:
+    tags: [ v* ]
```

Comment:
Comment:

Basicly this works fine, but we may have tag like 'valid-check' or 'void-something' in the future.
I mean that is it possible to use regex like `v[0-9]+.*` ?
I'm not decided to use 'v0.1-dev' or something, but i think it can be improved.

Reply:
> I mean that is it possible to use regex like `v[0-9]+.*` ?

Switched to use `v[0-9]+.*`. (I believe that should work.)

Reply:
Thank you so much! Good luck!